src/site/content/en/blog/building-a-breadcrumbs-component/index.md

```diff
@@ -0,0 +1,498 @@
+
+## Layouts
+
+### Markup
+
+Good components begin with appropriate HTML. In this next section I'll cover my
+markup choices and how they impact the overall component.
+
+#### Dark and light scheme
+
+```html
+<meta name="color-scheme" content="dark light">
+```
+
+{% Video
+  src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/I8w6oe28ueaZsDiOFssl.mp4",
+  class="w-screenshot",
+  autoplay="true",
+  loop="true",
+  muted="true"
+%}
+
+The [`color-scheme`](https://web.dev/color-scheme/) meta tag in the above
+snippet informs the browser that this page wants the light and dark browser
+styles. The example breadcrumbs don't include any CSS for these color schemes,
+and so the breadcrumbs will use the default colors provided by the browser.
+
+{% Aside %} Try removing all the styles from the demo page. The component works
+without them due to the well structured markup. {% endAside %}
+
+#### Navigation element
+
+```html
+<nav class="breadcrumbs" role="navigation"></nav>
+```
+
+It's appropriate to use the
+[`<nav>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element
+for site navigation. While the element has an implicit ARIA [role of
+navigation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role),
+in testing I noticed that having the attribute did indeed change the way a
+screen reader interacted with the element, it was actually announced as
+navigation, and so I've chosen to keep it there explicitly.
+
+#### Icons
+
+When an icon is repeated on a page, the SVG
+[`<use>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use) element
+means that you can define the `path` once, and use it for all instances of the
+icon. This prevents the same path information from being repeated, causing
+larger documents and the potential of path inconsistency. 
+
+To use this technique, add a hidden SVG element to the page and wrap the icons
+in a `<symbol>` element with a unique ID:
+
+```html
+<svg style="display: none;">
+
+  <symbol id="icon-home">
+    <title>A home icon</title>
+    <path d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6"/>
+  </symbol>
+  
+  <symbol id="icon-dropdown-arrow">
+    <title>A down arrow</title>
+    <path d="M19 9l-7 7-7-7"/>
+  </symbol>
+  
+</svg>
+```
+
+The browser reads the SVG HTML, puts the icon information into memory, and
+continues with the rest of the page referencing the ID for additional uses of
+the icon, like this:
+
+```html
+<svg viewBox="0 0 24 24" width="24" height="24" aria-hidden="true">
+  <use href="#icon-home" />
+</svg>
+
+<svg viewBox="0 0 24 24" width="24" height="24" aria-hidden="true">
+  <use href="#icon-dropdown-arrow" />
+</svg>
+```
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/BVkOZ3YnQBgF6K09sLTE.png",
+alt="devtools showing a rendered `<use>` SVG element", width="800", height="447"
```

Comment:
```suggestion
alt="DevTools showing a rendered `<use>` SVG element.", width="800", height="447"
```

Reply:
@rachelandrew Alt text should be full sentences with proper punctuation, reference here: https://developers.google.com/style/images?hl=en#alt-text

Reply:
nice, this is the 1st appearance of the alt text for her to review. i'll take note of this as well, since i do tend to put fragments into alt text, so i'll update my practices 👍🏻